dll/win32/shell32/droptargets/CFSDropTarget.cpp

```diff
@@ -180,6 +182,69 @@ BOOL CFSDropTarget::QueryDrop(DWORD dwKeyState, LPDWORD pdwEffect)
     return FALSE;
 }
 
+HRESULT CFSDropTarget::_GetEffectFromMenu(IDataObject *pDataObject, POINTL pt, DWORD *pdwEffect)
+{
+    HMENU hmenu = LoadMenu(shell32_hInstance, MAKEINTRESOURCEW(IDM_DRAGFILE));
+    if (!hmenu)
+        return E_OUTOFMEMORY;
+
+    /* FIXME: We need to support shell extensions here */
```

Comment:
👍 I was about to suggest that when I read the comment. :P

Reply:
Oh I should add it in the todo list

Reply:
What would this entail?